lib/ACLMicroservicesBuilder.js

```diff
@@ -229,42 +226,39 @@ ACLMicroservicesBuilder.prototype._executeSavePermissions = function(args, callb
 
   if(args.ent.id) { // update
 
-    self.ACLMicroservicesBuilder._loadAndAuthorize(entityDef, args.ent.id, args.cmd, roles, context, args.showSoftDenied$, function(err, dbEntity) {
+    self.ACLMicroservicesBuilder._loadAndAuthorize(entityDef, args.ent.id, args.cmd, roles, context, args.showSoftDenied$,args.ent, function(err, dbEntity) {
```

Comment:
not sure why you need to pass in the entity there. 
My understanding is that the bug is in the save_existing and the effect is that filters do not get run on save.
``_loadAndAuthorize`` is supposed to check if you have read access to that entity and executes all the 'load' ACLs. It should not need to know about the entity being saved. That is done further below.